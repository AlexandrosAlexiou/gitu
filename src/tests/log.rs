use super::*;

fn setup() -> TestContext {
    let ctx = TestContext::setup_clone();
    commit(ctx.dir.path(), "this-should-be-at-the-top", "");
    commit(ctx.dir.path(), "this-should-not-be-visible", "");
    ctx
}

#[test]
fn log_other_prompt() {
    snapshot!(setup(), "lljlo");
}

#[test]
fn log_other() {
    snapshot!(setup(), "lljlo<enter>");
}

#[test]
fn log_other_input() {
    snapshot!(setup(), "lomain~1<enter>");
}

#[test]
fn log_other_invalid() {
    snapshot!(setup(), "lo <enter>");
}
