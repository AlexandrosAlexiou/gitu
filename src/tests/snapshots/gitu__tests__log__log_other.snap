---
source: src/tests/log.rs
expression: ctx.redact_buffer()
---
â–Œ_______ add this-should-be-at-the-top                                          |
 _______ origin/main add initial-file                                           |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
                                                                                |
styles_hash: f31bd6c51e6ca60e
